packages/block-editor/src/components/button-block-appender/index.js

```diff
@@ -15,12 +15,13 @@ import { _x, sprintf } from '@wordpress/i18n';
 import BlockDropZone from '../block-drop-zone';
 import Inserter from '../inserter';
 
-function ButtonBlockAppender( { rootClientId, className } ) {
+function ButtonBlockAppender( { rootClientId, className, __experimentalSelectBlockOnInsert: selectBlockOnInsert } ) {
```

Comment:
The argument that `dispatch( 'core/block-editor' ).insertBlock` accepts is named `updateSelection`. Should we re-use that name for consistency?

Reply:
`<ButtonBlockAppender updateSelection />` didn't seem as clear to me as `<ButtonBlockAppender selectBlockOnInsert />`.